src/App.jsx

```diff
@@ -1,28 +1,32 @@
-import React,{Component} from 'react'
-import {BrowserRouter as Router, Route, Switch} from 'react-router-dom'
+import React from 'react'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
 import './css/style.css'
-import Home from './components/Home'
-import Forum from './components/Forum';
-import Events from './components/Events';
-import Resources from './components/Resources';
+import Home from './screens/Home'
+import Forum from './screens/Forum';
+import Events from './screens/Events';
+import Resources from './screens/Resources';
+import Header from './components/Header'
+import Footer from './components/Footer'
 
+const App = () => {
+    return (
+        <div id="body_container">
+            <Router>
+                <>
```

Comment:
Is this an empty component to only used to wrap the details? If so it looks good.

Reply:
Short syntax for React.Fragment, nice to learn new staff.

Reply:
Yeap, that is a short syntax for `<Fragment />`, and yes it's just wrapping because the Router requires just one child.

Reply:
Yay! Didn't know this existed. Thanks.

Reply:
I mean, didn't know about Fragment at all.